app/controllers/map_controller.rb

```diff
@@ -16,7 +16,10 @@ def index
   end
 
   def show
-    @node = DrupalNode.find_map_by_slug(params[:name]+'/'+params[:date])
+    @node = DrupalNode.find params[:name]
```

Comment:
@jywarren Actually friendly_id parameterizes the whole string. Slashes don't get well with parameterize. They get converted to dashes. Do you have any other format in mind? Though you can always use the default behavior friendly_id uses that appends a unique index at the en of slugs with similar title.